build/ReleaseManager/ReleaseManagerLib.py

```diff
@@ -58,9 +60,16 @@ def versionToRPMVersion(self, ver):
         return ver
 
     def createRPMReleaseNVR(self, rpmRel, rc):
-        nvr = "%s" % rpmRel
         if rc:
-            nvr = f"0.{rpmRel}.rc{rc}"
+            if rpmRel:
+                nvr = f"0.{rc}.rc{rc}.{rpmRel}"
+            else:
+                nvr = f"0.{rc}.rc{rc}"
```

Comment:
Just a suggestion. This could be written as:
```
nvr = f"0.{rc}.rc{rc}"
if rpmRel:
    nvr += f".{rpmRel}"
```

Reply:
Done. I changed also the line below similarly.